composables/gemini.ts

```diff
@@ -0,0 +1,45 @@
+const useGemini = () => {
+  const generateImage = async (prompt: string, imageUrl: string) => {
+    try {
+      // If the imageUrl is already a base64 string, use it directly
+      let base64Image = imageUrl;
+      if (!imageUrl.startsWith('data:')) {
+        // Otherwise fetch the image and convert to base64
+        const imageResponse = await fetch(imageUrl);
```

Comment:
the project uses no semis – i don't want to mess up your contribution history to this file but if u game change and i'll merge & deploy again @traf